sample-apps/react-native/dogfood/src/screens/Meeting/MeetingScreen.tsx

```diff
@@ -22,6 +24,7 @@ export const MeetingScreen = (props: Props) => {
 
   const onJoin = () => {
     setShow('active-call');
+    appStoreSetState({ chatLabelNoted: false });
```

Comment:
What is the idea behind having a global state for this? Local one should be enough as we reset it everytime